### DISCUSSION TITLE
Powering both Rapsberry Pi

### DISCUSSION THREAD

**Discussion by rrlevy** [tesla-android/issue-tracker#45] [Q&A]
How are you guys powering your Raspberries inside the car?

I'm having trouble making my setuyp work in a stable way. It crashes very often.

The most common problem is getting the "No Signal" message on the screen, for apparently no reasons.

I suspect it might have something to do with overheating or power delivery.

How are you powering your setups?

I tried using one port for each Raspberry, and I'm still having trouble. I was planning on using a 12V to USB adapter on the cigarette lighter port, but I want to check if anyone else is having problems with this.

**Comment by mbhforum** [comment]
I power my Rpi's with USB-C ports in my car using something like this:  https://www.amazon.com/Charger-AINOPE-Smallest-All-Metal-Compatible/dp/B09N9CSBC9/ref=sr_1_2_sspa?crid=L42Z2XKFLFV9&keywords=Aukey+USB+C+Car&qid=1654290101&sprefix=aukey+usb+c+c%2Caps%2C312&sr=8-2-spons&psc=1&spLa=ZW5jcnlwdGVkUXVhbGlmaWVyPUFRMEIwME44MDNXMjYmZW5jcnlwdGVkSWQ9QTAzNjg1NzQyT09UTTg5VjUzNlJSJmVuY3J5cHRlZEFkSWQ9QTAxNTA1NTYxM01PTlFYWEtaQVI3JndpZGdldE5hbWU9c3BfYXRmJmFjdGlvbj1jbGlja1JlZGlyZWN0JmRvTm90TG9nQ2xpY2s9dHJ1ZQ==

**Comment by ochompsky** [comment]
a double usb-c power plug in the 12v.

**Comment by rrlevy** [comment]
Thanks for the suggestions! I bought something similar and put it in this area beneath the armrest.

I'm still having issues with overheating... Trying to find the best way to deal with that
